lib/ui5Framework/maven/CacheMode.js

```diff
@@ -0,0 +1,18 @@
+
+
+/**
+ * Cache modes for maven consumption
+ *
+ * @public
```

Comment:
does `CacheMode` has to be public?

Reply:
Yes, since we reference it as a type in public JSDoc of for example the `graph` module